R/ide-snippet.R

```diff
@@ -0,0 +1,33 @@
+#' A Shiny app that can be used to construct a \code{spark_connect} statement
+#'
+#' @importFrom rstudioapi showQuestion
+#'
+#' @export
+#' @keywords internal
+#' @returns A Shiny app
+connection_databricks_shinyapp <- function() {
```

Comment:
I'm not sure if this is the right location, but would it be possible to pass in the cluster id in case `DATABRICKS_CLUSTER_ID` is not defined? If cluster id is provided, then the cluster id input box would be pre-populated and the connection snippet would be initialized immediately with the passed in cluster id value.

<img width="606" alt="image" src="https://github.com/mlverse/pysparklyr/assets/25834218/ee353280-f563-4ac9-bf95-66cae2ff2694">

Reply:
Discussed via DM -- instead of passing a parameter, the cluster id can be passed to the shiny app via environment variable `DATABRICKS_SELECTED_CLUSTER_ID`.

Reply:
The IDE can now pass `DATABRICKS_SELECTED_CLUSTER_ID` with the cluster ID selected. The app will use it as the default Cluster ID, and then clear the variable.